modules/angular2/src/services/url_resolver.ts

```
@@ -276,7 +276,13 @@ function _resolveUrl(base: string, url: string): string {
   var baseParts = _split(base);
 
   if (isPresent(parts[_ComponentIndex.SCHEME])) {
-    return _joinAndCanonicalizePath(parts);
+    var scheme = parts[_ComponentIndex.SCHEME];
+    if (scheme == 'package') {
+      var absPkgPath = '/node_modules/' + parts[_ComponentIndex.PATH];
```

Yes, please make this configurable via an `OpaqueToken`. Maybe use `@Optional` and throw if not set and `packages` is used, as  I doubt that people really serve their `node_modules`  folder in production...